### PR TITLE
refactor: adjusted resizable style and behaviors

### PR DESCRIPTION
**Pull request by MicroCBer**
Preview:

image

image

### CODE REVIEW

File: core/src/components/Resizable.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .ppd-resizable {
-  --inner-border-width: var(--border-width, 5px);
+  --inner-border-width: var(--border-width, 1px);
```
Comment (NWYLZW) [inline core/src/components/Resizable.scss:2]:
```suggestion
  --inner-border-width: var(--border-width, 5px);
```

Don't default value and I change the component caller props to change border width of `EditorZone` component.

File: core/src/components/Resizable.scss [hunk 2]
```diff
@@ -17,8 +17,12 @@
   }
   &-border {
     position: absolute;
-    background-color: #999;
-    transition: .1s;
```
Comment (NWYLZW) [inline core/src/components/Resizable.scss:21]: Why remove the transition?

**Review by NWYLZW — CHANGES_REQUESTED** [review]

File: core/src/components/Resizable.scss (hunk 2 above)
Comment (MicroCBer) [reply core/src/components/Resizable.scss:21]: Making the transformation instant can improve ux for smaller border width, making it easier to hold and drag.

**Review by NWYLZW — APPROVED** [review]
LGTM

**Pull request merged**